ScoutSuite/frontend/src/partials/aws/ec2.regions.id.vpcs.id.images.id/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { PartialValue } from '../../../components/Partial';
+import { partialDataShape } from '../../../utils/Partials';
+
+
+const propTypes = {
+  data: PropTypes.shape(partialDataShape).isRequired,
+};
+
+const Ec2Image = props => {
+  const { data } = props;
+
+  if (!data) return null;
+
+  // TODO: test this partial.
```

Comment:
Was is tested?

Reply:
there wasnt a finding to test it so i'll do it during beta when i'll test every findings to make sure they all work